src/models/returns.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+import Sequelize from 'sequelize';
+
+const {Model} = Sequelize;
+
+/**
+ * Build an Return model..
+ *
+ * @param {Sequelize.Sequelize} sequelize A Sequelize connection.
+ * @returns {Sequelize.Model} A returns model
+ * model.
+ */
+const ReturnsModel = (sequelize) => {
+  class Returns extends Model {}
+
+  Returns.init(
+    {
+      ApplicationId: {
+        type: Sequelize.INTEGER,
+        validate: {
+          notEmpty: true
+        }
+      },
+      beforeObjectiveRef: {
+        type: Sequelize.STRING
+      },
+      afterObjectiveRef: {
+        type: Sequelize.STRING
+      },
+      fromDate: {
+        type: Sequelize.DATE
+      },
+      toDate: {
+        type: Sequelize.DATE
+      },
+      comment: {
+        type: Sequelize.TEXT
+      },
```

Comment:
Are all these fields 'nullable', or should some of them be mandatory?

Reply:
resolved https://github.com/Scottish-Natural-Heritage/standard-forestry-operations-api/pull/30/files/0e442ea6c243eb5cf653655b019f9dfed092f0b6#diff-db980472dfb81403f7a047517b63b7987831d84c00883d6bcb68f1064fe2b295